source/FlexTable/FlexTable.js

```diff
@@ -44,7 +44,7 @@ export default class FlexTable extends Component {
     /** Optional CSS class to apply to all column headers */
     headerClassName: PropTypes.string,
     /** Fixed height of header row */
-    headerHeight: PropTypes.number.isRequired,
+    headerHeight: PropTypes.number,
```

Comment:
This should still be `.isRequired`. Your static property will handle the case when it's not passed in externally.

Reply:
I don't understand why the prop has to be marked as required if there is a default value.. can you elaborate?